pyEOS/eos.py

```diff
@@ -157,17 +157,29 @@ def load_candidate_config(self, filename=None, config=None):
         else:
             self.candidate_config.load_config(config=config)
 
-    def compare_config(self):
+    def compare_replace_config(self):
```

Comment:
if we leave it at "compare_config" it will be precisely the other way around than pyiosxr right now... that doesn't make me particularly happy, then again I guess I could change it in there. But I am not sure I agree to have: compare_merge_config() and compare_config() -- which is actually a replace